frontend/src/assets/css/scaffolding.scss

```
@@ -32,3 +33,29 @@ body {
   clip: rect(0 0 0 0);
   clip-path: inset(100%);
 }
+
+ul,
+ol,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
```

Margin is already reset for all elements upon `*` selector - add also `padding: 0` for all to reset
```suggestion
ul,
ol,
li {
  list-style: none;
}
```